pkg/cmd/service/service_test.go

```diff
@@ -590,8 +590,6 @@ func TestHTTPServingTLS(t *testing.T) {
 			KeyPath:  certsAndKeys.serverKeyFile,
 		}
 
-		fmt.Println(config)
```

Comment:
?

Reply:
I think I added this in on some PR way back when and now I am trying to sneak the deletion of it in this PR.